source/reference/access-mdb-shell-help.txt

```diff
@@ -242,3 +241,47 @@ Show Additional Usage Details for a Collection Method
 
 .. didn't include the cursor details from the manual help page as
    it seems as if most of it doesn't apply for ``mongosh``
+
+BSON Class Help
+---------------
+
+``mongosh`` provides help methods for :term:`BSON <BSON>` classes. The
+help methods provide a brief overview of the BSON class and a link with
+more information.
+
+To access help for BSON classes, run:
+
+.. code-block:: none
+
+   <BSON class>.help()
+
+For example, to view help for the ``ObjectId`` BSON class, run the
+following command:
+
+.. code-block:: javascript
+
+   ObjectId.help()
+
+``mongosh`` returns the following output:
+
+.. code-block:: javascript
+   :copyable: false
+
+   The ObjectId BSON Class:
+
+
```

Comment:
is there an extra return here?

Reply:
This is what the output looks like in the shell, but I think we can safely remove extra returns in the docs. Done.